sticker/lib/matrix.py

```diff
@@ -56,6 +56,7 @@ async def load_config(path: str) -> None:
             homeserver_url = config["homeserver"]
             access_token = config["access_token"]
             giphy_api_key = config["giphy_api_key"]
+            giphy_homeserver = config["giphy_homeserver"]
     except FileNotFoundError:
         print("Matrix config file not found. Please enter your homeserver and access token. Enter the Giphy API token if required, leave blank to disable the gif picker.")
```

Comment:
Should I ask the user for the homeserver that we use for MSC3860 here? It's currently only used if the config file already has the `giphy_homeserver` variable.

Reply:
Yeah could be prompted, and fill the default value if nothing is provided when prompting

Should probably call it `giphy_mxc_prefix` or something since it includes the `mxc://` prefix and trailing slash (as opposed to just the server name)

Reply:
I think `giphy_mxc_prefix` sounds good. Added some details about it as well.